tox.ini

```diff
@@ -95,3 +95,29 @@ setenv =
 commands =
     pip install -e .
 	behave --tags=-disabled integration_tests
+
+
+[testenv:ci_gsye_122]
+basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH
+deps = pip-tools
+whitelist_externals = git
+                      /bin/rm
+                      /bin/cp
+                      /bin/mkdir
+                      echo
+					  bash
```

Comment:
Do you really need to whitelist all of these?

Reply:
Probably not all of them, after addressing the use of a `envtmpdir`, and also previous comment on which we got rid of the cp ... line command.
I also forgot to get rid of `echo` which is not used at all. I'll fix this on the following commits .